### PR TITLE
[docs][notifications] Apply Android 13 advice to current/ previous SDK

### PR DESCRIPTION
**Pull request by keith-kurak**
# Why

I was still getting support questions referencing the old version of the Snack in the Notifications doc that I thought I fixed in https://github.com/expo/expo/pull/18949/commits. Noticed I didn't update the SDK 46 page!

# How

Copied unversioned to SDK 46 and 45 (other changes since 44 gave me pause on going back further, even though the same advise generally applies)

### CODE REVIEW

File: docs/pages/versions/v45.0.0/sdk/notifications.md [hunk 1]
```diff
@@ -839,7 +841,7 @@ An optional object of conforming to the following interface:
 }
 ```
 
-Each option corresponds to a different native platform authorization option (a list of iOS options is available [here](https://developer.apple.com/documentation/usernotifications/unauthorizationoptions), on Android all available permissions are granted by default and if a user declines any permission an app can't prompt the user to change).
+Each option corresponds to a different native platform authorization option. A list of iOS options is available [in the official documentation](https://developer.apple.com/documentation/usernotifications/unauthorizationoptions). On Android, all available permissions are granted by default, and if a user declines any permission, an app cannot prompt the user to change.
```
Comment (amandeepmittal) [inline docs/pages/versions/v45.0.0/sdk/notifications.md:844]:
```suggestion
Each option corresponds to a different native platform authorization option. To see a list of available options on iOS, see  [UNAuthorizationOptions](https://developer.apple.com/documentation/usernotifications/unauthorizationoptions). On Android, all available permissions are granted by default, and if a user declines any permission, an app cannot prompt the user to change.
```

File: docs/pages/versions/v45.0.0/sdk/notifications.md [hunk 2]
```diff
@@ -278,13 +280,13 @@ async function registerForPushNotificationsAsync() {
 
 ## Custom notification icon and colors (Android only)
 
-In the managed workflow, set your [`notification.icon`](../config/app.md#notification) and [`notification.color`](../config/app.md#notification) keys in **app.json**, rebuild your app, and you're good to go!
+[Expo Prebuild](/workflow/prebuild) users can configure the [`notification.icon`](../config/app.md#notification) and [`notification.color`](../config/app.md#notification) keys in the project's **app.json** or by using the [`expo-notifications` config plugin directly](#optional-setup). These are build-time setting so you'll need to recompile your native Android app with `eas build -p android` or `npx expo run:android` to see the changes.
```
Comment (amandeepmittal) [inline docs/pages/versions/v45.0.0/sdk/notifications.md:283]:
```suggestion
You can configure  the [`notification.icon`](../config/app.md#notification) and [`notification.color`](../config/app.md#notification) keys in the project's **app.json** if you are using [Expo Prebuild](/workflow/prebuild) or by using the [`expo-notifications` config plugin directly](#optional-setup). These are build-time settings, so you'll need to recompile your native Android app with `eas build -p android` or `npx expo run:android` to see the changes.
```

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
Thanks for updating this doc! The changes look good to me 👍 

I left some minor suggestions.

**Comment by keith-kurak** [comment]
> Thanks for updating this doc! The changes look good to me 👍
> 
> I left some minor suggestions.

These all look good to me. I think these should apply these to 46 and unversioned, as well? This PR was to sync the changes to unversioned to the recent SDK version docs. I can take care of this.

**Comment by keith-kurak** [comment]
@amandeepmittal heads-up I just synced your tweaks to all 46 and unversioned

**Comment by keith-kurak** [comment]
@amandeepmittal apologies for the confusion... it looked like you did request changes to all versions, but somehow I didn't see them the first time. At any rate, I copied your same changes to 46 and unversioned, so I think it's good :-)

**Review by amandeepmittal — APPROVED** [review]
@keith-kurak, no worries! I should have mentioned the changes for SDK 46 too. 

LGTM 👍

**Pull request merged**